agent/main.go

```diff
@@ -268,6 +269,8 @@ func hostApi(socketHarOutputChannel chan<- *tapApi.OutputChannelItem) {
 	routes.EntriesRoutes(app)
 	routes.MetadataRoutes(app)
 	routes.StatusRoutes(app)
+	routes.InstallRoutes(app)
```

Comment:
are these routes for "standalone mode" only?

Reply:
They're only relevant there, yes.
We could make them conditional but there's no real reason to, the only endpoint there does nothing when not in standalone mode

Reply:
lets hide same as here
https://github.com/up9inc/mizu/blob/bf686892123755b16eea4eece8bc393a2a76c080/agent/main.go#L263